third_party/terraform/utils/node_config.go.erb

```diff
@@ -20,240 +20,240 @@ var defaultOauthScopes = []string{
+					Computed:         true,
+					ForceNew:         true,
+					// Legacy config mode allows explicitly defining an empty taint.
+					// See https://www.terraform.io/docs/configuration/attr-as-blocks.html
+					ConfigMode: schema.SchemaConfigModeAttr,
+					Elem: &schema.Resource{
+						Schema: map[string]*schema.Schema{
+							"key": {
+								Type:     schema.TypeString,
+								Required: true,
+								ForceNew: true,
+							},
+							"value": {
+								Type:     schema.TypeString,
+								Required: true,
+								ForceNew: true,
+							},
+							"effect": {
+								Type:         schema.TypeString,
+								Required:     true,
+								ForceNew:     true,
+								ValidateFunc: validation.StringInSlice([]string{"NO_SCHEDULE", "PREFER_NO_SCHEDULE", "NO_EXECUTE"}, false),
+							},
 						},
 					},
 				},
-			},
 
-			"workload_metadata_config": {
-<% if version.nil? || version == 'ga' -%>
-				Removed: "This field is in beta. Use it in the the google-beta provider instead. See https://terraform.io/docs/providers/google/guides/provider_versions.html for more details.",
-				Computed: true,
-<% end -%>
-				Type:       schema.TypeList,
-				Optional:   true,
-				ForceNew:   true,
-				MaxItems:   1,
-				Elem: &schema.Resource{
-					Schema: map[string]*schema.Schema{
-						"node_metadata": {
-							Type:         schema.TypeString,
-							Required:     true,
-							ForceNew:     true,
-							ValidateFunc: validation.StringInSlice([]string{"UNSPECIFIED", "SECURE", "EXPOSE", "GKE_METADATA_SERVER"}, false),
+				"workload_metadata_config": {
+	<% if version.nil? || version == 'ga' -%>
+					Removed: "This field is in beta. Use it in the the google-beta provider instead. See https://terraform.io/docs/providers/google/guides/provider_versions.html for more details.",
+					Computed: true,
+	<% end -%>
+					Type:       schema.TypeList,
+					Optional:   true,
+					ForceNew:   true,
+					MaxItems:   1,
+					Elem: &schema.Resource{
+						Schema: map[string]*schema.Schema{
+							"node_metadata": {
+								Type:         schema.TypeString,
+								Required:     true,
+								ValidateFunc: validation.StringInSlice([]string{"UNSPECIFIED", "SECURE", "EXPOSE", "GKE_METADATA_SERVER"}, false),
+							},
 						},
 					},
 				},
-			},
 
-			"sandbox_config": {
-<% if version.nil? || version == 'ga' -%>
-				Removed: "This field is in beta. Use it in the the google-beta provider instead. See https://terraform.io/docs/providers/google/guides/provider_versions.html for more details.",
-				Computed: true,
-<% end -%>
-				Type:       schema.TypeList,
-				Optional:   true,
-				ForceNew:   true,
-				MaxItems:   1,
-				Elem: &schema.Resource{
-					Schema: map[string]*schema.Schema{
-						"sandbox_type": {
-							Type:         schema.TypeString,
-							Required:     true,
-							ValidateFunc: validation.StringInSlice([]string{"gvisor"}, false),
+				"sandbox_config": {
```

Comment:
We removed ForceNew from `sandbox_config` but the code seems to only apply to workload_metadata_config, how does that work?

Reply:
i for sure removed it from the wrong config when transferring over to MM